src/components/DetailView/DetailView.svelte

```diff
@@ -76,6 +78,18 @@
       close.focus();
     }
   });
+
+  // Reference to the vega chart component.
+  let vegaRef = null;
+
+  function downloadVega() {
+    vegaRef.vegaAccessor().then((view) => {
+      const pngP = view.toImageURL('png', 2);
+      pngP.then((url) => {
+        downloadUrl(url, `${sensor.name} in ${'Unknown'}.png`);
```

Comment:
since the detail view is showing one or more regions, can you add them to the file name instead of the 'Unknown'?

Reply:
Right... I forgot that I had simplified this because I wasn't able to get the names for some reason.  I'll figure that out.

Reply:
It's not clear how I can get the region names.  Your previous code generated the name with:
`${sensor.name} in ${$currentRegionInfo ? $currentRegionInfo.displayName : 'Unknown'}.png`
 but currentRegionInfo is not available in the version I started from.  Adding it to the imports and using the same string seems to work.

Reply:
thx. while this will work for the single region case, it won't create a good title for a comparison case. Can you take a look how the h4 title tag a few lines below is generated. It should give you some hints to handle both cases in a generic way.

see also https://deploy-preview-556--cmu-delphi-covidcast.netlify.app/?sensor=doctor-visits-smoothed_adj_cli&level=county&date=20201007&signalType=value&encoding=color&mode=overview&region=42003&compare=37111

Reply:
OK, I should have noticed that.  I made the change, staged, committed, and pushed it, but I don't see it here yet.

Reply:
Hmm, I added a comment here that seems to have disappeared.  It said I made the change to the filename, staged, committed, and pushed it, but I don't see any evidence of it here.

Reply:
Ah, there is my earlier comment now.  And maybe the push didn't happen because ... the VSCode UI is very confusing, but I see it now.

Reply:
Github notices you that it is now an outdated conversation on the code: 
![image](https://user-images.githubusercontent.com/4129778/95751923-8f649880-0c9f-11eb-809c-66ee3e3b5a8a.png). 

The idea is that when you provide a review to a certain line of code but the line changes in a later commit it will remain there but warning you that the code you are viewing is outdated.